demo/index.html

```diff
@@ -31,40 +31,30 @@
         able to expand the underlying CSS custom properties.
         See: https://github.com/Polymer/polymer-modulizer/issues/154
         -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
-
-$_documentContainer.innerHTML = `<custom-style><style is="custom-style" include="demo-pages-shared-styles">
-    .centered {
-      min-width: 600px;
-    }
-    demo-snippet {
-      --demo-snippet-code: {
-        max-height: 250px;
-      }
-    }
-  </style></custom-style>`;
+  <script type="module">
```

Comment:
not possible to fix this case either? if the `ShadyCSS.styleDocument()` suggestion from dan isn't working, maybe we should ask him again?

Reply:
I couldn't get this one to work either. I'm guessing it's because `@polymer/iron-demo-helpers/demo-pages-shared-styles.js` hasn't been run before this `<custom-style>`; putting this back in HTML directly causes a warning to show up in the console: "Could not find style data in module named demo-pages-shared-styles". The `ShadyCSS.styleDocument` call doesn't seem to have any effect.